### ISSUE TITLE
Storage for inactive screens

### ISSUE THREAD

**Issue by cjoohs** [opinsys/iivari#20]
It would be nice to have a solution for occasionally used screens. For example a separate storage for them could be a solution. There can be screens that are needed only once a year, so the rest of the time those screens are just filling the screen list for nothing. It's also not nice to re-write same info again every time in case you delete the rarely used screen.
